core/src/main/java/io/kestra/core/models/tasks/NamespaceFilesInterface.java

```diff
@@ -0,0 +1,14 @@
+package io.kestra.core.models.tasks;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public interface NamespaceFilesInterface {
+    @Schema(
+        title = "Inject namespace files",
+        description = "Inject namespace file on this task. If true, inject all namespaces files or use filter to limit injected files.",
+        anyOf = {Boolean.class, NamespaceFiles.class}
+    )
+    @PluginProperty
+    Object getNamespaceFiles();
```

Comment:
Wouldn't it be easier to use the `NamespaceFiles` object here and have a way to inject everything from it?
Like a `all` boolean or just document that to inject everything one should configure `include: **/*`?

I wonder if those multi-types that we often use are easy to use for not too technical people.